packages/nc-gui/composables/useViewData.ts

```diff
@@ -66,6 +59,13 @@ export function useViewData(
 
   const { isUIAllowed } = useUIPermission()
 
+  const formatData = (list: Record<string, any>[]) =>
+    list.map((row) => ({
+      row: { ...row },
+      oldRow: { ...row },
+      rowMeta: {},
+    }))
```

Comment:
Why was this moved inside the composable? 
Outside it would only be created once, inside as many times as the composable is called and this function doesn't seem to have dependencies that require it to be inside